content/course-specific-material/introR-NatGW-Aug16.md

```diff
@@ -1,11 +1,15 @@
 ---
+author: 
 date: 2016-08-29
 slug: introR-NatGW-Aug16
+draft: True
 title: National GW Workshop
-menu: 
-  main:
-    parent: Course Specific Material
-    weight: 1
+categories: Data Science
+ 
+tags: 
+  - R
+ 
+ 
```

Comment:
Yeah, the workflow here is kind of strange. Delete all of the lines that got added (empty author, draft: TRUE, categories, and tags). Then add the menu: stuff back in. That's needed for the collapsible menu on the sidebar